llvm/lib/Target/M68k/M68kISelLowering.cpp

```diff
@@ -204,11 +204,12 @@ M68kTargetLowering::getExceptionSelectorRegister(const Constant *) const {
   return M68k::D1;
 }
 
-unsigned
+InlineAsm::ConstraintCode
 M68kTargetLowering::getInlineAsmMemConstraint(StringRef ConstraintCode) const {
-  return StringSwitch<unsigned>(ConstraintCode)
-      .Case("Q", InlineAsm::Constraint_Q)
-      .Case("U", InlineAsm::Constraint_Um) // We borrow Constraint_Um for 'U'.
+  return StringSwitch<InlineAsm::ConstraintCode>(ConstraintCode)
+      .Case("Q", InlineAsm::ConstraintCode::Q)
+      .Case("U",
+            InlineAsm::ConstraintCode::Um) // We borrow Constraint_Um for 'U'.
```

Comment:
nit: "We borrow Constraint::Um for 'U'"

Reply:
good catch! done in https://github.com/llvm/llvm-project/pull/66003/commits/a7691b807a95a1640f8ee153109513a37e0f61fa.